web/src/__snapshots__/ErrorPane.test.tsx.snap

```diff
@@ -83,7 +122,14 @@ exports[`renders one container start error 1`] = `
         </p>
       </header>
       <section>
-        Eeeeek there is a problem
+        <code>
+          <span
```

Comment:
...I'm confused by the `className={null}` etc. 🤔

Reply:
Me too!

but the existing snapshots for build errors had them, so I chose not to worry about it.